packages/visualizations/addon/templates/components/cell-renderers/threshold.hbs

```diff
@@ -1,2 +1,2 @@
-{{!-- Copyright 2017, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
-{{metricValue}}
+{{!-- Copyright 2018, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+{{format-table-cell metricValue format}}
```

Comment:
Now, would it be better to extend the metric cell renderer for this? 
@jkusa what do you think?

Reply:
@balajimanoharan maybe, can `threshold` extend `metric`?

Reply:
it probably could. there only one extra property in threshold, `valueIndicator`. Everything else is the same.

Reply:
@jkusa updated

Reply:
Do we need the helper now?

Reply:
@jkusa Do you refer to the template file?